src/view.js

```diff
@@ -58,86 +76,125 @@ export class View {
     const gameRuleParagraph = this.#createElement(
       'p',
       'Guess the interval between the 2 tones.'
     );
 
-    const buttonsGridContainer = this.#createElement('div');
-
-    const submitAndMoveToNextQuestionButton =
-      this.#createButton('Move to next');
-    submitAndMoveToNextQuestionButton.disabled = true;
-    const skipQuestionButton = this.#createButton('Skip');
-
-    const currentScoreDisplay = this.#createElement('div', 'Score: ');
-    const currentScoreSpan = this.#createElement('span');
-    currentScoreSpan.id = 'currentScore';
-    currentScoreDisplay.append(currentScoreSpan);
-
-    this.appContainer.append(
-      currentQuestionNumberDisplay,
-      playTonesButton,
-      gameRuleParagraph,
-      buttonsGridContainer,
-      skipQuestionButton,
-      submitAndMoveToNextQuestionButton,
-      currentScoreDisplay
-    );
-
+    // Answer: interval buttons, guess button, skip button
     const intervalButtons = Object.entries(intervals).map(
       ([intervalName, semitones]) => {
         const button = this.#createButton(intervalName);
+        button.classList.add(classNames.secondaryButton);
+        button.dataset.semitones = semitones;
         button.addEventListener('click', () => {
+          this.#resetSelectedInterval();
           this.#currentSelectedIntervalSemitones = semitones;
+          this.#toggleIntervalButtonState(semitones);
           submitAndMoveToNextQuestionButton.disabled = false;
         });
         return button;
       }
     );
 
+    const submitAndMoveToNextQuestionButton = this.#createButton('Guess');
+    submitAndMoveToNextQuestionButton.classList.add(classNames.primaryButton);
+    submitAndMoveToNextQuestionButton.disabled = true;
+    const skipQuestionButton = this.#createButton('Skip');
+    skipQuestionButton.classList.add(classNames.tertiaryButton);
+    const buttonsGridContainer = this.#createElement('div');
+    buttonsGridContainer.classList.add(classNames.buttonsContainer);
     buttonsGridContainer.append(...intervalButtons);
+    buttonsGridContainer.append(submitAndMoveToNextQuestionButton);
+    buttonsGridContainer.append(skipQuestionButton);
+
+    // Update app
+    this.appContainer.append(
+      headerContainer,
+      playTonesButton,
+      gameRuleParagraph,
+      buttonsGridContainer
+    );
 
+    // Event listeners
     playTonesButton.addEventListener('click', () => {
       const now = Tone.now();
-      this.#sampler.triggerAttackRelease(this.#currentNote1, '8n', now);
-      this.#sampler.triggerAttackRelease(this.#currentNote2, '8n', now + 1);
+      this.#sampler.triggerAttackRelease(
+        this.#currentNote1,
+        this.#toneLength,
+        now
+      );
+      this.#sampler.triggerAttackRelease(
+        this.#currentNote2,
+        this.#toneLength,
+        now + 1
+      );
       this.isPlayTonesButtonClicked = true;
+      playTonesButton.disabled = true;
+      setTimeout(
```

Comment:
Oh I love this idea that while tones are being played, a user can't click the button again! So smart and elegant idea